lib/mediaprocessor/VisualRecognition.js

```diff
@@ -99,6 +97,7 @@ function detectFacesVR(filename) {
     } catch (e) {
       reject(e);
     }
+    log.debug('calling visualRecognition detectFaces', params);
```

Comment:
did you want this?

Reply:
not really. was hoping it would help isolate what call was generating the error